contracts/AaveLenderBorrowerCloner.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "./Strategy.sol";
+
+contract AaveLenderBorrowerCloner {
+    address public immutable original;
+
+    event Cloned(address indexed clone);
+    event Deployed(address indexed original);
+
+    constructor(
+        address _vault,
+        address _yVault,
+        bool _isWantIncentivised,
+        bool _isInvestmentTokenIncentivised,
```

Comment:
it's weird that this contructor doesn't match the one in the strategy. AFAIK there is a second call which would `setStrategyParams`  after a clone, so I would get rid of those two bools here and default is false.

Reply:
second call would not be required if we are OK with default params

having those bools incorrectly set might cause problems so I would leave it like it is atm